x-pack/plugins/security_solution/public/common/components/ml_popover/hooks/use_security_jobs_helpers.tsx

```diff
@@ -159,5 +166,5 @@ export const createSecurityJobs = (
   const installedJobs = getInstalledJobs(jobSummaryData, moduleSecurityJobs, compatibleModuleIds);
```

Comment:
Commenting here so it's threaded, but looks like the new space-aware jobs are being classified as `Custom jobs`. I threw a breakpoint in the call to `getAugmentedFields()` (line 66/67) and am seeing them not match because of the new space prefix in this check:

```const moduleJob = moduleJobs.find((mj) => mj.id === jobId)```

<img width="659" alt="image" src="https://user-images.githubusercontent.com/2946766/220765859-a0e325ee-b734-4f18-bbbb-ac218bdf3fcb.png">

<p align="center">
  <img width="500" src="https://user-images.githubusercontent.com/2946766/220766061-d386a582-de46-4e01-be96-7bda3c8b77f1.gif" />
</p>

Reply:
Nice catch! 🐞🐛!
We must use `matchJobId` whenever comparing a module job id with an installed job id.